server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -1654,13 +1655,12 @@ public Host updateHost(final UpdateHostCmd cmd) throws NoTransitionException {
                 }
             }
         }
-
-        final List<String> hostTags = cmd.getHostTags();
+        List<String> hostTags = cmd.getHostTags();
```

Comment:
@DK101010 why removed 'final' ?

Reply:
Arrrr... my bad, I removed this in my first version. but then I found a better way and forgot the add this again.